### PR TITLE
add resx to validation

### PR DESCRIPTION
**Pull request by AnnPostnikova**
https://trello.com/c/eRGxTTEm

### CODE REVIEW

File: src/ProjectService.Validation/Project/Resources/ProjectRequestValidationResource.ru-RU.resx [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+    <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <data name="AddUserTwice" xml:space="preserve">
+    <value>Пользователь не может быть добавлен в проект дважды</value>
+  </data>
+  <data name="CustomerLong" xml:space="preserve">
+    <value>Наименование заказчика проекта слишком длинное</value>
+  </data>
+  <data name="DepartmentIdIsEmpty" xml:space="preserve">
+    <value>Id департамента не может быть пустым</value>
```
Comment (MariyaSin) [inline src/ProjectService.Validation/Project/Resources/ProjectRequestValidationResource.ru-RU.resx:127]: remove

File: src/ProjectService.Validation/Project/Resources/ProjectRequestValidationResource.ru-RU.resx [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+  </data>
+  <data name="EndDateUtcIsNull" xml:space="preserve">
+    <value>Если проект закрыт, у него должна быть дата закрытия</value>
+  </data>
+  <data name="IncorrectStatus" xml:space="preserve">
+    <value>Указан некорретный статус проекта</value>
```
Comment (MariyaSin) [inline src/ProjectService.Validation/Project/Resources/ProjectRequestValidationResource.ru-RU.resx:133]: remove

File: src/ProjectService.Validation/Project/Resources/ProjectRequestValidationResource.ru-RU.resx [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+  </data>
+  <data name="NameExists" xml:space="preserve">
+    <value>Проект с таким именем уже существует</value>
+  </data>
+  <data name="NameLong" xml:space="preserve">
+    <value>Имя проекта слишкои длинное</value>
```
Comment (MariyaSin) [inline src/ProjectService.Validation/Project/Resources/ProjectRequestValidationResource.ru-RU.resx:139]: remove

File: src/ProjectService.Validation/Project/Resources/ProjectRequestValidationResource.ru-RU.resx [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+  </data>
+  <data name="NameNotNullOrEmpty" xml:space="preserve">
```
Comment (MariyaSin) [inline src/ProjectService.Validation/Project/Resources/ProjectRequestValidationResource.ru-RU.resx:141]: remove

File: src/ProjectService.Validation/Project/Resources/ProjectRequestValidationResource.ru-RU.resx [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+    <value>Имя не может быть пустым</value>
+  </data>
+  <data name="ShortDescriptionLong" xml:space="preserve">
+    <value>Краткое описание проекта слишком длинное</value>
```
Comment (MariyaSin) [inline src/ProjectService.Validation/Project/Resources/ProjectRequestValidationResource.ru-RU.resx:145]: remove

File: src/ProjectService.Validation/Project/Resources/ProjectRequestValidationResource.ru-RU.resx [hunk 6]
```diff
@@ -0,0 +1,159 @@
...
+  </data>
+  <data name="ShortNameExists" xml:space="preserve">
+    <value>Проект с таким сокращенным именем уже существует</value>
+  </data>
+  <data name="ShortNameLong" xml:space="preserve">
+    <value>Сокращенное имя проекта слишком длинное</value>
```
Comment (MariyaSin) [inline src/ProjectService.Validation/Project/Resources/ProjectRequestValidationResource.ru-RU.resx:151]: remove

File: src/ProjectService.Validation/Project/Resources/ProjectRequestValidationResource.ru-RU.resx [hunk 7]
```diff
@@ -0,0 +1,159 @@
...
+  </data>
+  <data name="UserIdIsEmpty" xml:space="preserve">
+    <value>Id пользователя не может быть пустым</value>
```
Comment (MariyaSin) [inline src/ProjectService.Validation/Project/Resources/ProjectRequestValidationResource.ru-RU.resx:154]: remove

File: src/ProjectService.Validation/Project/Resources/ProjectRequestValidationResource.resx [hunk 8]
```diff
@@ -0,0 +1,159 @@
...
+    <value>2.0</value>
+  </resheader>
+  <resheader name="reader">
+    <value>System.Resources.ResXResourceReader, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <resheader name="writer">
+    <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <data name="AddUserTwice" xml:space="preserve">
+    <value>Пользователь не может быть добавлен дважды</value>
```
Comment (MariyaSin) [inline src/ProjectService.Validation/Project/Resources/ProjectRequestValidationResource.resx:121]: RU?

**Review by MariyaSin — APPROVED** [review]

File: src/ProjectService.Validation/Project/Resources/ProjectRequestValidationResource.resx [hunk 9]
```diff
@@ -0,0 +1,159 @@
...
+    <value>Project short name already exists</value>
+  </data>
+  <data name="ShortNameLong" xml:space="preserve">
+    <value>Project short name is too long</value>
+  </data>
+  <data name="UserIdIsEmpty" xml:space="preserve">
+    <value>User id must not be empty</value>
+  </data>
+  <data name="UsersDoNotExist" xml:space="preserve">
+    <value>Some users does not exist</value>
```
Comment (ndoriomedov) [inline src/ProjectService.Validation/Project/Resources/ProjectRequestValidationResource.resx:157]: don't

**Review by ndoriomedov — APPROVED** [review]

**Pull request merged**